.github/workflows/deployment-test.yml

```diff
@@ -39,6 +40,8 @@ jobs:
         with:
           node-version: 14.x
           cache-built: false
+        env:
+          XSNAP_RANDOM_INIT: '1'
```

Comment:
since XS and xsnap don't document this, it might be worth a short comment here about what this does, somebody grepping widely enough might find it